packages/graphback-codegen-client/src/index.ts

```diff
@@ -1,2 +1,5 @@
 export * from './ClientDocuments'
-export * from './ClientCRUDPlugin'
\ No newline at end of file
+export * from './ClientCRUDPlugin'
+
+// Required for plugins
+export { ClientCRUDPlugin as Plugin } from './ClientCRUDPlugin'
```

Comment:
I wonder is `Plugin` too generic. `ClientCRUDPlugin as GraphbackPlugin`?

Reply:
The purpose for this is to be generic. This is how we know that package defines plugin. Better docs needed.